packages/client-react-streaming/src/ManualDataTransport/ManualDataTransport.tsx

```diff
@@ -121,6 +121,55 @@ const buildManualDataTransportBrowserImpl =
 
 const UNINITIALIZED = {};
 
+/**
+ * > This export is only available in React Client Components
+ *
+ * Creates a "manual" Data Transport, to be used with `WrapApolloProvider`.
+ *
+ * @remarks
+ *
+ * ### Drawbacks
+ *
+ * While this Data Transport enables streaming SSR, it has some conceptual drawbacks:
+ *
+ * - It does not have a way of keeping your connection open if your application already finished, but there are still ongoing queries that might need to be awaited transported over.
+ *   - This can happen if a component renders `useBackgroundQuery`, but does not read the `queryRef` with `useReadQuery`
+ *   - These "cut off" queries will be restarted in the Browser once the browser's `load` event happens
+ * - If the `useInsertHtml` doesn't immediately flush data to the browser, the browser might already attain "newer" data through queries triggered by user interaction.
+ *   - This delayed behaviour is the case with the Next.js `ServerInsertedHTMLContext` and in the example Vite implementation.
+ *   - In this, case, older data from the server might overwrite newer data in the browser.
+ *   - This is minimized by simulating ongoing queries in the browser once the information of a started query is transported over.
```

Comment:
```suggestion
 *   - In this, case, older data from the server might overwrite newer data in the browser. This is minimized by simulating ongoing queries in the browser once the information of a started query is transported over.
```

The 2nd sentence feels more like a continuation of the third, not so much its own point, so I think combining these makes sense.